app/src/main/java/swati4star/createpdf/util/WhatsNewUtils.java

```diff
@@ -21,6 +21,10 @@
 
 public class WhatsNewUtils {
 
+    private WhatsNewUtils(){
```

Comment:
make the other functions of this non static
Otherwise, there is no point in making it singleton.